FileShares/src/FileSharesWeb/appsettings.Development.json

```diff
@@ -0,0 +1,45 @@
+{
+  // Steeltoe: Add schema to get auto-completion.
+  "$schema": "https://steeltoe.io/schema/v4/schema.json",
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  // Steeltoe: Configure health endpoint to show components and details.
+  "Management": {
+    "Endpoints": {
+      "Health": {
+        "ShowComponents": "Always",
+        "ShowDetails": "Always"
```

Comment:
```suggestion
        "ShowDetails": "Always",
        "DiskSpace": {
          "Path": "\\\\localhost\\steeltoe_network_share"
        }
```

For https://github.com/SteeltoeOSS/Steeltoe/pull/1503. And then `FileShareHealthContributor` can be removed.

Reply:
This would work locally when the instructions are followed as-is but does not account for what happens when its deployed, so I used PostConfigure to set the path based on what is in `FileShareConfiguration` instead.

Reply:
Screenshot of the health check in TAS (I manually set ShowComponents and ShowDetails to get this image)
![image](https://github.com/user-attachments/assets/77634569-6874-4c0e-a94a-42b7cfd444bc)

Reply:
PostConfigure is even better. Nice to see it work in TAS, looks like I don't need to test that anymore.